apps/ewallet_db/test/ewallet_db/api_key_test.exs

```diff
@@ -61,6 +61,32 @@ defmodule EWalletDB.APIKeyTest do
       insert(:wallet).address,
       insert(:wallet).address
     )
+
+    test "update enable state with `expired`" do
+      {:ok, api_key} = APIKey.insert(params_for(:api_key))
+      assert api_key.enabled == true
+
+      {:ok, api_key} = APIKey.update(api_key, %{"expired" => true})
+      assert api_key.enabled == false
+    end
+  end
+
+  describe "enable_or_disable/2" do
```

Comment:
Can you add a test that disabling it twice doesn't make it enabled?

Just in case someone turns it into a toggle in the future.